WeWatch/WeWatch/Service/Webservice.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  Webservice.swift
+//  WeWatch
+//
+//  Created by Anton on 30/11/2024.
+//
+
+import Foundation
+
+enum AuthenticationError: Error {
+    case invalidCredentials
+    case custom (errorMessage: String)
+}
+
+struct LoginRequestBody: Codable {
+    let apikey: String
+    let pin: String
+}
+
+struct LoginResponse: Codable {
+    let status: String
+    let data: LoginData?
+}
+
+struct LoginData: Codable {
+    let token: String?
+}
+
+class Webservice {
+    func login(apikey: String, pin: String, completion: @escaping (Result<String, AuthenticationError>) -> Void) {
```

Comment:
Figure out how to make this function more universal. So that you can send requests not only to login, but also to other endpoints.